flatcar.pkr.hcl

```
@@ -84,7 +84,7 @@ build {
       "chmod +x flatcar-install",
 
       # Install flatcar
-      "./flatcar-install -v -s -o hetzner -b ${var.image_base_url}/${local.board[source.name]} -V ${var.image_version_id}",
+      "./flatcar-install -v -s -o hetzner -b https://${var.channel}.release.flatcar-linux.net/${local.board[source.name]} -V ${var.image_version_id}",
```

Is there any downside to just pass the channel + board to the install script and let it build the url itself?

```suggestion
      "./flatcar-install -v -s -o hetzner -C ${var.channel} -B ${local.board[source.name]} -V ${var.image_version_id}",
```

---

Ah yes, it's even better to do like this. I was focus on the URL :D